compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -1130,8 +1130,18 @@ fn should_encode_trait_impl_trait_tys(tcx: TyCtxt<'_>, def_id: DefId) -> bool {
 }
 
 // Return `false` to avoid encoding impl trait in trait, while we don't use the query.
```

Comment:
This comment needs to be modified to specify that it's specifically encoding the *new* RPITIT lowering (to assoc types).

Reply:
Done